src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -1254,43 +1294,58 @@ public static boolean unsolvedAndNotSimple(MethodCallExpr method) {
 
   /**
    * For a method call that is not simple, this method will take that method as input and create
-   * corresponding synthetic class
+   * corresponding synthetic class.
    *
-   * @param method the method call to be taken as input
+   * @param methodCall the method call to be taken as input
```

Comment:
This string needs to be a particular format. I think it needs to be a static invocation: in particular, it must not have a receiver, since you use a substring call to find the first `(` and expect the result to be the beginning of the arguments list. If there was a receiver like the following, this would fail:
"foo.getBar().methodCall(...)" - if we passed this whole string, the substring call below will only find "foo.getBar", not "foo.getBar().methodCall", as we intend.

So, please give the exact specification that you expect here in this Javadoc.